release.mk

```diff
@@ -751,6 +756,21 @@ $O/$(ECLIPSE_VANILLA_LINUX_TGZ):
 $O/$(ECLIPSE_VANILLA_WIN_ZIP):
 	$(WGET) $(WGETFLAGS) -O $@ '$(ECLIPSE_DL_LINK_BASE)/$(ECLIPSE_VANILLA_WIN_ZIP)&r=1'
 
+#
+# Building IDE Plugins
+#
+ifeq ($(ENABLE_IDE_PLUGINS_BUILD),y)
+
+BUILD_DIR=arc_gnu_eclipse/build
+OUTPUT_DIR=arc_gnu_eclipse/repository/target/
```

Comment:
Those variables names are too ambigous and might clash with other compnents. They also should be declated close other variables, here https://github.com/foss-for-synopsys-dwc-arc-processors/toolchain/blob/arc-dev/release.mk#L299

You'd need something like this:
```Makefile
IDE_PLUGIN_SRC_DIR=$(ROOT)/arc_gnu_eclipse
IDE_PLUGIN_BUILD_DIR=$(IDE_PLUGIN_SRC_DIR)/build
IDE_PLUGIN_OUT_DIR=$(IDE_PLUGIN_SRC_DIR)/build/repository/target/
```

Although BUILD_DIR and OUTDIR, might not be useful at all, because each is used only once.